libfwupdplugin/fu-udev-device.h

```diff
@@ -40,7 +39,6 @@ typedef enum {
 	FU_UDEV_DEVICE_FLAG_NONE = 0,
 	FU_UDEV_DEVICE_FLAG_OPEN_READ = 1 << 0,
 	FU_UDEV_DEVICE_FLAG_OPEN_WRITE = 1 << 1,
-	FU_UDEV_DEVICE_FLAG_VENDOR_FROM_PARENT = 1 << 2,
 	FU_UDEV_DEVICE_FLAG_OPEN_NONBLOCK = 1 << 4,
 	FU_UDEV_DEVICE_FLAG_OPEN_SYNC = 1 << 5,
 	FU_UDEV_DEVICE_FLAG_IOCTL_RETRY = 1 << 6,
```

Comment:
don't forget to slide these up (unless you have another to go in another PR)

Reply:
And it's gone!